### DISCUSSION TITLE
Latest beta Apple Music error

### DISCUSSION THREAD

**Discussion by Kebo2r** [music-assistant/support#2648] [Q&A]
downloaded the beta yesterday.  Now getting this error when trying to play Apple Music:

2024-07-20 07:28:53.597 ERROR (MainThread) [music_assistant.webserver] Error handling message: player_queues/play_pause: ClientSession._request() got an unexpected keyword argument 'encoded'

Worked fine in the previous couple of betas.  Have deleted, reinstalled, reconfigured, etc.  Local Media works.

Thanks!

**Comment by OzGav** [comment]
@MarvinSchenkel

**Reply by MarvinSchenkel** [reply]
@marcelveldt this does not sound like an error that's related to Apple Music specifically. Did anything change in rc3 related to play/pause?

**Comment by Kebo2r** [comment]
FYI.  I just added the RadioBrowser and it plays fine from that as well.  Thanks for calls help and work!  Can’t wait to get MA going fully.  Let me know if I can help any other way.

**Reply by MarvinSchenkel** [reply]
Maybe you can move this to a github issue and include a debug log? That always helps us tracking down an issue more quickly.

**Comment by Kebo2r** [comment]
Done.  Log attached.

https://github.com/music-assistant/hass-music-assistant/issues/2649￼
Apple Music error in RC3 · Issue #2649 · music-assistant/hass-music-assistant
github.com
